{"metadata": {"dbt_schema_version": "https://schemas.getdbt.com/dbt/sources/v3.json", "dbt_version": "1.0.3", "generated_at": "2022-03-01T02:39:42.083905Z", "invocation_id": "48e3b88b-b3cf-4f83-9db3-644dae92f9a0", "env": {}}, "results": [{"unique_id": "source.coffee_shop.coffee_shop.orders", "max_loaded_at": "2021-05-31T00:00:28+00:00", "snapshotted_at": "2022-03-01T02:39:40.650086+00:00", "max_loaded_at_time_ago_in_s": 23683152.650086, "status": "error", "criteria": {"warn_after": {"count": null, "period": null}, "error_after": {"count": 1, "period": "hour"}, "filter": null}, "adapter_response": {}, "timing": [{"name": "compile", "started_at": "2022-03-01T02:39:40.254527Z", "completed_at": "2022-03-01T02:39:40.254533Z"}, {"name": "execute", "started_at": "2022-03-01T02:39:40.254849Z", "completed_at": "2022-03-01T02:39:41.467356Z"}], "thread_id": "Thread-2", "execution_time": 1.2138445377349854}, {"unique_id": "source.coffee_shop.coffee_shop.customers", "error": "Database Error in source customers (models/sources.yml)\n  Unrecognized name: created_at at [3:11]", "status": "runtime error"}], "elapsed_time": 2.588548183441162}